### ISSUE TITLE
Summary of input errors

### ISSUE THREAD

**Issue by bewards** [victorjonsson/jQuery-Form-Validator#551]
I've been using the inline error message callback (`inlineErrorMessageCallback`) for full support of WCAG (accessibility) by mapping error messages to inputs using aria-describedby and aria-invalid=true/false, seen below:

```
inlineErrorMessageCallback: function ($input, errorMessage, config) {
    var inputName = $input.attr('name') || $input.attr('id');
    var fieldErrorStr = inputName + '_err';

    createErrorPopup($input, errorMessage, fieldErrorStr);

    return false;
}
```

```
function createErrorPopup($input, errorMessage, fieldErrorStr) {

    // get message container: check input attribute then fallback to parent
    var $msgContainer = helper.exists($input.attr('data-validation-error-msg-container')) ?
        $($input.attr('data-validation-error-msg-container')) : $input.parent();

    if (errorMessage) {
        // build span.help-block.form-error#id, and append to container
        var msg = '<span class="help-block form-error" id="' + fieldErrorStr + '">' + errorMessage + '</span>';

        $msgContainer.append(msg);
        $input.attr('aria-describedBy', fieldErrorStr);
    } else {
        $msgContainer.find('.form-error').remove();
    }
}
```

For some of my other forms, I need to list out a Summary of the errors below the form because they do not fit inline. Just as the inline logic, It would make sense to map each error msg to the invalid input with aria-describedby. Currently using the callback `submitErrorMessageCallback`, I noticed that all required fields that contain errors are merged into a single error message:

![image](https://cloud.githubusercontent.com/assets/3927169/21745151/b2e429ee-d4f2-11e6-9913-4f0521684eb9.png)

It doesn't look like this callback can map error messages to invalid inputs, infact it looks like I should be using the inline callback and append all those error messages to a container for an outcome like below. And do some token replacement for required-field types to get the "First Name" value from the placeholder attribute or whatnot into the error message:

![image](https://cloud.githubusercontent.com/assets/3927169/21745192/971d4fe6-d4f3-11e6-8920-e452a0e493cc.png)
